schemas/esper-study-telemetry.1.schema.json

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
@Dexterp37 Is this good enough you think? :)

Reply:
Good enough :-) I left my comments on the [related PR](https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/98#pullrequestreview-78512394).